### PR TITLE
Memberdeletedevent handler  projection

### PR DESCRIPTION
**Pull request by iiNomad23**

### CODE REVIEW

File: ClubService.Application/EventHandlers/MemberEventHandlers/MemberDeletedEventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    public async Task Handle(DomainEnvelope<IDomainEvent> domainEnvelope)
+    {
+        if (!Supports(domainEnvelope))
+        {
+            return;
+        }
+        
+        var memberReadModel = await memberReadModelRepository.GetMemberById(domainEnvelope.EntityId);
+        
+        if (memberReadModel != null)
```
Comment (Smightym8) [inline ClubService.Application/EventHandlers/MemberEventHandlers/MemberDeletedEventHandler.cs:21]: Do reduce nesting we could change the if statement to `if (memberReadModel == null) return;`

**Review by Essiga — CHANGES_REQUESTED** [review]
don't lock

**Review by Smightym8 — APPROVED** [review]
lgtm

**Review by Essiga — APPROVED** [review]
kiitos

**Pull request merged**